src/index.js

```diff
@@ -19,6 +19,15 @@ import DataExplorer from './containers/DataExplorer';
 
 const store = configureStore();
 
+import messages from './messages'; // Lang does not know where did you get your messages from.
+
+import LangSugar from './lang';
+window.L = new LangSugar();
```

Comment:
`window` makes me nervous here. It's fine for now, but I think something like language could definitely be on react's `context`. I don't know enough gotchas about doing it that way, but we should make it a TODO after the alpha.